tests/TestCase/TestSuite/IntegrationTestTraitTest.php

```diff
@@ -69,6 +71,16 @@ public function setUp(): void
             $routes->options('/options/:controller/:action', []);
             $routes->connect('/:controller/:action/*', []);
         });
+        Router::scope('/cookie-csrf/', ['csrf' => 'cookie'], function (RouteBuilder $routes) {
+            $routes->registerMiddleware('cookieCsrf', new CsrfProtectionMiddleware());
```

Comment:
I wonder if we should rename this to CookieCsrfProtectionMiddleware.

Reply:
We could do that, we'd need an alias for backwards compatibility.